src/leader_scheduler.rs

```diff
@@ -295,6 +306,18 @@ impl LeaderScheduler {
         self.get_scheduled_leader(tick_height).map(|(id, _)| id)
     }
 
+    // Give a total tick count, returns 1) the block index and 2) the number of ticks in that block
+    pub fn get_block_and_index(&self, tick_height: u64) -> (u64, u64) {
```

Comment:
all this stuff could be made simpler if bootstrap_height was a multiple of blocks, yes?

Reply:
...

Reply:
oops, yup, I think that's true!